financial-templates-lib/logger/ConsoleTransport.js

```diff
@@ -0,0 +1,22 @@
+// This transport enables Winston logging to the console.
+const winston = require("winston");
+
+const alignedWithColorsAndTime = winston.format.combine(
+  winston.format.colorize(),
+  winston.format.timestamp(),
+  winston.format.printf(info => {
+    const { timestamp, level, ...args } = info;
+
+    const ts = timestamp.slice(0, 19).replace("T", " ");
```

Comment:
What's the significance of the `19`?

Reply:
it formats the time stamp to look like this `2020-03-24 13:27:08`

Reply:
ok, just add a comment showing example output